portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -65,78 +65,80 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-   ArrayList<Message> tasks = new ArrayList<>();
+   ArrayList<Comment> tasks = new ArrayList<>();
 
     for (Entity entity : results.asIterable()) {
-    Message newMessage = new Message((Key) entity.getKey(),
-                                        (String) entity.getProperty("text"),
-                                        (long) entity.getProperty("timestamp"));
+
+      Key uniqueKey = (Key) entity.getKey();
+      String comment = (String) entity.getProperty("comment");
+      long timestamp = (long) entity.getProperty("timestamp");
+      String picture = (String) entity.getProperty("imageUrl");
```

Comment:
The thing is called imageUrl both in datastore entity and in message to be serialized to JSON. It makes sense to name the variable imageUrl as well instead of picture for consistency.